### PR TITLE
Feedback

### PR DESCRIPTION
**Pull request by poglet**
hi @poglet please check x,y

### CODE REVIEW

**Comment by poglet** [comment]
Looking at the feedback branch

File: prac_02/word_generator.py [hunk 1]
```diff
@@ -10,7 +10,7 @@
 VOWELS = "aeiou"
 CONSONANTS = "bcdfghjklmnpqrstvwxyz"
 
-word_format = "ccvcvvc"
+word_format = "cccvc"
```
Comment (poglet) [inline prac_02/word_generator.py]: Unsure why this was changed

File: prac_05/hex_codes.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+             'pale': '#db7093',
+             'pink': '#ffc0cb',
+             'powderblue': '#b0e0e6',
+             'purple': '#a020f0',
+             'white': '#ffffff',
+             'whitesmoke': '#f5f5f5'}
+for colour, hexnumber in HEX_CODES.items():
+    print('{:<18} is {}'.format(colour, hexnumber))
+
+colour = (input('Enter a colour to find its HEX code: ')).lower()
```
Comment (lindsaymarkward) [inline prac_05/hex_codes.py]: Remove redundant parentheses.

File: prac_05/hex_codes.py [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+while colour != "":
+    if colour in HEX_CODES:
+        print(colour, 'is', HEX_CODES[colour])
```
Comment (lindsaymarkward) [inline prac_05/hex_codes.py]: Use {} string formatting

File: prac_05/state_names.py [hunk 4]
```diff
@@ -4,15 +4,21 @@
...
 """
 
-# TODO: Reformat this file so the dictionary code follows PEP 8 convention
-STATE_NAMES = {"QLD":"Queensland", "NSW": "New South Wales", "NT" :"Northern Territory", "WA" : "Western Australia", "ACT":"Australian Capital Territory", "VIC":"Victoria", "TAS":"Tasmania"}
+# TODO: I've reformatted the disctionary and made it fit within with line (pep8)
+STATE_NAMES = {"QLD": "Queensland", "NSW": "New South Wales", "NT": "Northern Territory", "WA": "Western Australia",
+               "ACT": "Australian Capital Territory", "VIC": "Victoria", "TAS": "Tasmania"}
+
 # print(STATE_NAMES)
+for short_state, long_state in STATE_NAMES.items():
```
Comment (lindsaymarkward) [inline prac_05/state_names.py]: 👍

File: prac_05/state_names.py [hunk 5]
```diff
@@ -4,15 +4,21 @@
...
+    print('{:<3} is {}'.format(short_state, long_state))
 
-state = input("Enter short state: ")
+state = (input("Enter short state: ")).upper()
```
Comment (lindsaymarkward) [inline prac_05/state_names.py]: Why are you doing this? I figured it was a mistake, but now it seems on purpose.
`x.y()` doesn't need to become `(x).y()`; it makes sense (syntax is fine) as it is.

File: prac_05/word_counter.py [hunk 6]
```diff
@@ -0,0 +1,20 @@
+longest_word = 0
+count_dict = {}
+
+text = input('Text: ')
+text = (text.split(" "))
```
Comment (lindsaymarkward) [inline prac_05/word_counter.py]: More redundant parentheses!

File: prac_05/word_counter.py [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+
+for word in text:
+    count_dict[word] = count_dict.get(word, 0) + 1
+    if len(word) > longest_word:
+        longest_word = len(word)
```
Comment (lindsaymarkward) [inline prac_05/word_counter.py]: Would you expect a variable called `longest_word` to store a number, or a ... word?

File: prac_05/word_counter.py [hunk 8]
```diff
@@ -0,0 +1,20 @@
...
+
+
+"""
+This gets the values from the dictionary into a list, sorted by key value.
+The list if formatted to support the longest word.
+"""
+sorted_list = []
+sorted_list.extend((sorted(count_dict, key=count_dict.get)))
```
Comment (lindsaymarkward) [inline prac_05/word_counter.py]: You love (these) don't you :)

Why are you using `extend`? Just use =

**Pull request merged**